v3/how-to-guides/01-basics/d-genesis-config/index.mdx

```diff
@@ -5,85 +5,113 @@ keywords: basics, beginner, runtime
 version: '3.0'
 section: how to guides
 category: basics
-difficulty: 1
 ---
 
-<Message
-  type={`yellow`}
-  title={`Information`}
-  text={`This is intended for beginners who want to become familiar with customizing their chain. Not all configurations will follow the same approach. This is the most basic approach. Learn more in the [related material section](#related-material).`}
-/>
-
-<Objectives
-  data={[
-    {
-      title: 'Goal',
-      description: `Learn how to customize a chain's genesis configuration for the [balances pallet](/rustdocs/latest/pallet_balances/index.html).`,
-    },
-    {
-      title: 'Use Cases',
-      description: `Initialize balances for endowed accounts.`,
-    },
-    {
-      title: 'Overview',
-      description: `Genesis configuration is a useful tool for testing chain behavior by defining an initial state for things such as accounts, 
-        balances, genesis for custom pallets, and more. Here is a simple guide on how to configure custom initial balances for a runtime, 
-        by modifying [BalancesConfig](/rustdocs/latest/node_template_runtime/type.BalancesConfig.html) in \`chainspec.rs\`.`,
-    },
-  ]}
-/>
-
-## Steps
-
-### 1. Modify accounts
-
-In `chain_spec.rs`, modify the accounts-to-amount map to apply it to the set of all endowed accounts (this is how every node template is set up):
+Genesis configuration is a tool for testing chain behavior by defining an initial state for things such as accounts, balances, genesis for custom pallets, and more. 
```

Comment:
"tool" is a kind of strange word to me.

I might call it a "property" of the chain, or a "configuration" of the chain.